Foodie/order.rb

```diff
@@ -0,0 +1,77 @@
+class Order
+  attr_accessor :items, :order_id, :customer, :status
+  @@menu={
+  "PAVBHAJI"=>70,"DOSA"=>30,"CHHOLE BHATURE"=>80,"PULAV"=>60,
+  "PIZZA"=>99,"PASTA"=>65,"MANCHURIAN"=>70,"BURGER"=>50,
+  "UTTAPPA"=>35,"HAKKA NOODLES"=>65,"PANEER CHILLI"=>80
+  }
+  @@order_count=0
+
+  def Order.order_count
+    @@order_count
+  end
+  
+  def take_order
+    self.order_id=@@order_count+1
+    print "Customer Name: "
+    customer = gets.chomp.upcase
+    if customer.match(/^[A-Z]+\s?[A-Z]*\s?[A-Z]*$/)
```

Comment:
use early return instead of having big if else loop. 

something like
`p 'Invalid Name !!!' and return unless customer.match(/^[A-Z]+\s?[A-Z]*\s?[A-Z]*$/)`